### PR TITLE
Update module initializers feature status

### PR DESCRIPTION
**Pull request by jnm2**

### CODE REVIEW

File: docs/Language Feature Status.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ efforts behind them.
...
-| [Module initializers](https://github.com/RikkiGibson/csharplang/blob/module-initializers/proposals/module-initializers.md) | TBD                                                                                    | [In progress / design](https://github.com/dotnet/roslyn/issues/40500)                                         | [RikkiGibson](https://github.com/RikkiGibson) [jnm2](https://github.com/jnm2)| [AlekseyTs](https://github.com/AlekseyTs)                                          | [gafter](https://github.com/gafter) |
+| [Module initializers](https://github.com/dotnet/csharplang/blob/master/proposals/module-initializers.md) | [features/module-initializers](https://github.com/dotnet/roslyn/tree/features/module-initializers)                                                                                    | [In progress / design](https://github.com/dotnet/roslyn/issues/40500)                                         | [RikkiGibson](https://github.com/RikkiGibson) [jnm2](https://github.com/jnm2)| [AlekseyTs](https://github.com/AlekseyTs)                                          | [gafter](https://github.com/gafter) |
```
Comment (jnm2) [inline docs/Language Feature Status.md:27]: https://github.com/dotnet/csharplang/blob/master/proposals/module-initializers.md: https://github.com/dotnet/csharplang/pull/3392 merged.

https://github.com/dotnet/roslyn/tree/features/module-initializers: https://github.com/dotnet/roslyn/pull/43281 merged.

**Comment by RikkiGibson** [comment]
@jnm2 we typically base updates to this document on `master`

**Comment by jnm2** [comment]
@RikkiGibson Thanks, keep not even seeing that option when I create the PR.

**Comment by RikkiGibson** [comment]
@jnm2 please force push to the branch or create a new PR to prevent merging module-initializers implementation changes to master.

**Comment by jnm2** [comment]
@RikkiGibson Updated.

**Pull request merged**